web/public/gameData_v1.0-23.json

```diff
@@ -10221,7 +10633,377 @@
                 "power": 4
             },
             "isAlternate": false,
-            "isFicsmas": false
+            "isFicsmas": false,
+            "isPowerGenerator": false
+        }
+    ],
+    "powerGenerationRecipes": [
```

Comment:
By the looks of it, the results of this key need to not use the Recipes interface but a new one with similar attributes. There's a lot of redundant / obsolete  data in here.